mappings/net/minecraft/server/network/ConnectionListener.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/class_9 net/minecraft/server/network/ConnectionListener
+	FIELD field_28 LOGGER Ljava/util/logging/Logger;
+	FIELD field_29 open Z
+	FIELD field_30 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_31 socket Ljava/net/ServerSocket;
+	FIELD field_32 thread Ljava/lang/Thread;
+	FIELD field_33 connectionCounter I
+	FIELD field_34 pendingConnections Ljava/util/ArrayList;
+		COMMENT ArrayList of ServerLoginNetworkHandler
+	FIELD field_35 connections Ljava/util/ArrayList;
+		COMMENT ArrayList of ServerPlayNetworkHandler
```

Comment:
Sometimes times I see the word play. Is it a shortened form of player or is it actually related to play? Is this just the way of modern fabric?

Reply:
It's related to the different network phases introduced in 13w41a. There's distinct betekent handlers for each phase. Before then, this wasn't necessarily the case (hence `ClientNetworkHandler`) but server network handlers are split between login and normal play.

Reply:
Ah that makes sense. Good to know! Also nice to see you here in the reviews! 🥳
Looking forward to the day when we fully integrate with Ornithe

Reply:
As am I! It will be a while still, we've some issues to work out yet.

Reply:
♥️, no worries take your time. Enjoy the process 🙂

Reply:
hype for gen2!!!